src/sentry/incidents/grouptype.py

```diff
@@ -27,29 +35,136 @@ class MetricIssueEvidenceData(EvidenceData):
     def create_occurrence(
         self,
         evaluation_result: ProcessedDataConditionGroup,
         data_packet: DataPacket[QuerySubscriptionUpdate],
         priority: DetectorPriorityLevel,
-    ) -> tuple[DetectorOccurrence, dict[str, Any]]:
-        # Returning a placeholder for now, this may require us passing more info
-        occurrence = DetectorOccurrence(
-            issue_title="Some Issue Title",
-            subtitle="An Issue Subtitle",
-            type=MetricIssue,
-            level="error",
-            culprit="Some culprit",
+    ) -> tuple[DetectorOccurrence, EventData]:
+        try:
+            alert_rule_detector = AlertRuleDetector.objects.get(detector=self.detector)
+            alert_id = alert_rule_detector.alert_rule_id
+        except AlertRuleDetector.DoesNotExist:
+            alert_id = None
+
+        try:
+            detector_trigger = DataCondition.objects.get(
+                condition_group=self.detector.workflow_condition_group, condition_result=priority
+            )
+        except DataCondition.DoesNotExist:
+            raise DetectorException(
+                f"Failed to find detector trigger for detector id {self.detector.id}, cannot create metric issue occurrence"
+            )
+
+        try:
+            query_subscription = QuerySubscription.objects.get(id=data_packet.source_id)
+        except QuerySubscription.DoesNotExist:
+            raise DetectorException(
+                f"Failed to find query subscription for detector id {self.detector.id}, cannot create metric issue occurrence"
+            )
+
+        try:
+            snuba_query = SnubaQuery.objects.get(id=query_subscription.snuba_query_id)
+        except SnubaQuery.DoesNotExist:
+            raise DetectorException(
+                f"Failed to find snuba query for detector id {self.detector.id}, cannot create metric issue occurrence"
+            )
+
+        try:
+            assignee = parse_and_validate_actor(
+                str(self.detector.created_by_id), self.detector.project.organization_id
+            )
+        except Exception:
+            assignee = None
+
+        title = self.construct_title(snuba_query, detector_trigger, priority)
+        event_data = {
+            "project_id": self.detector.project_id,
+            "environment": self.detector.config.get("environment"),
+            "platform": None,
+            "sdk": None,
+        }  # XXX: may need to add to this
+
+        return (
+            DetectorOccurrence(
+                issue_title=self.detector.name,
+                subtitle=title,
+                resource_id=None,
+                evidence_data={
+                    "detector_id": self.detector.id,
+                    "value": priority,
+                    "data_condition_ids": [detector_trigger.id],
+                    "data_condition_type": detector_trigger.type,
+                    "data_condition_comparison_value": self.extract_value(data_packet),
```

Comment:
This is another one of those values that we have a lot done in the platform: https://github.com/getsentry/sentry/blob/master/src/sentry/workflow_engine/handlers/detector/stateful.py#L483-L504 🤔 it's def not super clear _what_ we are going to provide and what is expected to be set.

are there any tools you use in your editor that would help give hints on this too?

(just curious so we can simplify this for the next detector)